packages/flutter_tools/lib/src/aot.dart

```diff
@@ -175,7 +175,6 @@ class AotBuilder {
       case TargetPlatform.android_arm64:
       case TargetPlatform.android_x86:
       case TargetPlatform.darwin_x64:
-        return true;
```

Comment:
The code below the `if(_canUseAssemble() ...)` above in the `AotBuilder` constructor is also dead with this. Maybe include a comment there like the one below if that is intentional.

Reply:
Done